Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SeedingAlgorithm.hpp

```diff
@@ -94,6 +98,32 @@ class SeedingAlgorithm final : public IAlgorithm {
       m_inputSpacePoints{};
 
   WriteDataHandle<SimSeedContainer> m_outputSeeds{this, "OutputSeeds"};
+
+  static inline bool ItkFastTrackingCuts(float bottomRadius, float cotTheta) {
```

Comment:
```suggestion
  static inline bool itkFastTrackingCuts(float bottomRadius, float cotTheta) {
```

or `applyItkFastTrackingCuts`